assets/js/amp-customize-controls.js

```diff
@@ -1,19 +1,27 @@
 ( function( api, $ ) {
 	'use strict';
 
+	var ampVars = window.ampVars;
+
+	api.state.add( 'ampEnabled', new api.Value() ).set( false );
+	api.state.add( 'ampAvailable', new api.Value() ).set( false );
+
 	/**
 	 * Check if the URL is AMPified.
 	 *
 	 * @param {string} url URL.
 	 * @return {boolean} whether it is an AMP URL.
 	 */
 	function isAmpUrl( url ) {
-		var urlParser = document.createElement( 'a' );
+		var urlParser = document.createElement( 'a' ),
+			regexParam = new RegExp( "(^|\\?|&)" + ampVars.query + "=1(?=&|$)" ),
+			regexEndpoint = new RegExp( "\\/" + ampVars.query + "\\/?$" );
+
 		urlParser.href = url;
-		if ( /(^|\?|&)amp=1(?=&|$)/.test( urlParser.search ) ) {
+		if ( regexParam.test( urlParser.search ) ) {
```

Comment:
Instead of `regexParam` you can make use of `wp.customize.utils.parseQueryString()` and do something like:

```js
if ( ! _.isUndefined( wp.customize.utils.parseQueryString( urlParser.search.substr( 1 ) )[ ampVars.query ] ) ) {
```